server/webapp/WEB-INF/rails/webpack/views/pages/pipelines_as_code.tsx

```diff
@@ -106,17 +111,23 @@ export class PipelinesAsCodeCreatePage extends Page {
             onchange={() => {
               if (syncConfigRepoWithMaterial()) {
                 this.material(cloneMaterialForPaC(vm.material));
+                this.configRepo().material(this.material());
               }
             }}
           />
 
           <MaterialEditor material={this.material()} hideTestConnection={syncConfigRepoWithMaterial()} scmOnly={true} showLocalWorkingCopyOptions={false} disabled={syncConfigRepoWithMaterial()}/>
         </UserInputPane>
         <div class={css.verifyDefsInMaterial}>
+          <IdentifierInputField label="Identifier" property={this.configRepo().id} errorText={this.configRepo().errors().errorsForDisplay("id")} />
```

Comment:
I think we've found that users aren't clear about "Identifier" as it sounds like something we should generate, not them. I know this is basically a primary key used to reference a configrepo, but for other similar concepts, we call this a "name" (e.g., pipeline name). Could we change this label to read "Name this repository"? Or similar?

Reply:
@marques-work the only issue is the validation error will say `id must be present` 

We can change the error message in the constructor, but that'll effect other forms as well.

Reply:
@ibnc Hmm. Does this error come from the config repo model? specifically, `validatePresenceOf()`. Let's make the change to the label, and I'll look to see where we might need to fix the validation message elsewhere